packages/itwinui-react/src/core/SideNavigation/SideNavigation.tsx

```diff
@@ -110,14 +130,28 @@ export const SideNavigation = React.forwardRef((props, forwardedRef) => {
       {...rest}
```

Comment:
I think the `...rest` props should pass into the `iui-side-navigation` element and then create a `wrapperProps` prop for this element, similar to what was done in the [RadioTile PR](https://github.com/iTwin/iTwinUI/pull/1409/files). Thoughts? @mayank99

Reply:
i'm not sure. In the case `RadioTile`, the `<input>` was the main element, but in this case they are both regular divs (neither is more important than the other).

cc @gretanausedaite

Reply:
Would that be within the scope of this PR, or should that be covered by a separate one?

Reply:
If we are keeping current code, I'd like wrapperProps more than innerProps.

> maybe we need to break this down into subcomponents: `SideNavigation.Wrapper` and `SideNavigation`

Is there a use case when these two are not together?

Reply:
> If we are keeping current code, I'd like wrapperProps more than innerProps.

ok, lets go with `wrapperProps` then.

---

> > maybe we need to break this down into subcomponents: `SideNavigation.Wrapper` and `SideNavigation`
> 
> Is there a use case when these two are not together?

i think the user might to omit the wrapper if they already have a parent with `position: relative`. it would be similar to info-panel.

Reply:
I've never created + implemented a new subcomponent within a component before, so it's hard for me to gauge if such a change exceeds the scope of this PR.

Reply:
maybe you misread the thread. the consensus is to go with simple `wrapperProps`, without any subcomponents.

Reply:
Done